examples/layers/widgets/styled_text.dart

```diff
@@ -100,12 +98,13 @@ class _StyledTextDemoState extends State<StyledTextDemo> {
           mainAxisAlignment: MainAxisAlignment.center,
           crossAxisAlignment: CrossAxisAlignment.start,
           children: _kNameLines
-            .map<Widget>((List<String> nameAndText) => _toText(nameAndText[0], nameAndText[1]))
-            .expand((Widget line) => <Widget>[
-              line,
-              const SpeakerSeparator(),
-            ])
-            .toList()..removeLast(),
+              .map<Widget>((List<String> nameAndText) => _toText(nameAndText[0], nameAndText[1]))
+              .expand((Widget line) => <Widget>[
```

Comment:
Not sure why we're suddenly indenting by four spaces here. IMHO the original version was easier to read.

Reply:
Line continuation is 4-spaces indent with dartfmt. On Flutter codebase we have a lot of places where a 4-spaces is  used like this.